crates/editor/src/editor.rs

```diff
@@ -11535,6 +11538,7 @@ impl ViewInputHandler for Editor {
     fn selected_text_range(&mut self, cx: &mut ViewContext<Self>) -> Option<Range<usize>> {
         // Prevent the IME menu from appearing when holding down an alphabetic key
         // while input is disabled.
+        #[cfg(not(target_os = "linux"))]
```

Comment:
What's going on with this change?

Reply:
While in vim mode the line below makes selected_text_range return None.
And the IME panel gets stuck at the position we entered vim mode.

I could make a new function.

Reply:
I find it strange that this line specifically would cause the problem. It looks like it's just checking if the input is enabled or disabled... and if it's disabled this short circuit is correct.